--- src/tools_webrtc/apple/copy_framework_header.py.orig	2025-09-10 00:00:00
+++ src/tools_webrtc/apple/copy_framework_header.py	2025-09-10 00:00:00
@@ -20,7 +20,7 @@
   if not match:
     return line
 
-  return '%s#import <WebRTC/%sRTC%s.h>%s' % (match.group(1), match.group(3),
+  return '%s#import <WebRTCLib/%sRTC%s.h>%s' % (match.group(1), match.group(3),
                                              match.group(4), match.group(5))
 
 
