--- src/sdk/objc/native/src/objc_video_encoder_factory.mm	2025-09-01 12:15:08
+++ src/sdk/objc/native/src/objc_video_encoder_factory.mm	2025-09-04 00:35:01
@@ -16,6 +16,9 @@
 #import "base/RTCVideoEncoder.h"
 #import "base/RTCVideoEncoderFactory.h"
 #import "components/video_codec/RTCCodecSpecificInfoH264+Private.h"
+#ifdef RTC_ENABLE_H265
+#import "components/video_codec/RTCCodecSpecificInfoH265+Private.h"
+#endif
 #import "sdk/objc/api/peerconnection/RTCEncodedImage+Private.h"
 #import "sdk/objc/api/peerconnection/RTCVideoCodecInfo+Private.h"
 #import "sdk/objc/api/peerconnection/RTCVideoEncoderSettings+Private.h"
@@ -67,7 +70,14 @@
                                     class]]) {
           codecSpecificInfo = [(RTC_OBJC_TYPE(
               RTCCodecSpecificInfoH264) *)info nativeCodecSpecificInfo];
+        }
+#ifdef RTC_ENABLE_H265
+        else if ([info isKindOfClass:[RTC_OBJC_TYPE(RTCCodecSpecificInfoH265)
+                                        class]]) {
+          codecSpecificInfo = [(RTC_OBJC_TYPE(
+              RTCCodecSpecificInfoH265) *)info nativeCodecSpecificInfo];
         }
+#endif
 
         EncodedImageCallback::Result res =
             callback->OnEncodedImage(encodedImage, &codecSpecificInfo);
