--- src/tools_webrtc/ios/generate_umbrella_header.py.orig	2025-09-10 00:00:00
+++ src/tools_webrtc/ios/generate_umbrella_header.py	2025-09-10 00:00:00
@@ -41,7 +41,7 @@
     */\n\n""" % datetime.datetime.now().year))
 
     for s in args.sources:
-      outfile.write("#import <WebRTC/{}>\n".format(os.path.basename(s)))
+      outfile.write("#import <WebRTCLib/{}>\n".format(os.path.basename(s)))
 
   return 0
